presto-docs/src/main/sphinx/release/release-0.233.rst

```diff
@@ -0,0 +1,79 @@
+=============
+Release 0.233
+=============
+
+General Changes
+_______________
+* Fix an optimizer failure introduced in ``0.229``, where a ``LIKE`` pattern is deduced into a constant, e.g., ``col LIKE 'a' and col = 'b'``.
+* Fix ``ArbitraryOutputBuffer`` to avoid skewing output data distribution.
+* Fix function registration for function :func:`classification_fall_out`.
+* Add configuration property ``exchange.async-page-transport-enabled`` to turn on non-blocking IO for page transport,
+  and add configuration property ``exchange.async-page-transport-timeout`` to set for server side timeout.
+  Use URL prefix ``/v1/task/async`` for async page transport request.
+* Add configuration property ``task.http-timeout-concurrency`` to improve performance of task timeout executors.
+* Add support for soft affinity scheduling, which makes the best effort to fetch the same piece of data from the same worker,
+  while allowing fallback to random workers if the preferred workers are too busy to handle additional splits.
+  The feature can be turned on by ``ConnectorSplit#getNodeSelectionStrategy()``.
+* Add soft affinity scheduling to bucketed split. It makes the best effort to fetch the same bucket data from the same worker.
+  When using dynamic group scheduling, if the preferred workers are unavailable to handle the specific bucket splits, it will fallback to random workers.
```

Comment:
Should we refer to the connectors documentation?  It will be missed in release notes

Reply:
@kewang1024 Could you please update the connector documentation? Have a high-level description in the release notes and link to the documentation.

Reply:
* Add support for soft affinity scheduling for unbucketed split. Refer to connectors documentation.
* Add soft affinity scheduling to bucketed split. Refer to connectors documentation.